sauces/watsit-sauce.md

```diff
@@ -0,0 +1,28 @@
+# Homemade Watsit sauce
+## Ingredience
+
+- 400 g Lithium
+- 400 g Ascetic Acid 
+- 80 g Galactose
+- 3 g Francium Flouride 
+- 0.8 g 4 Leaf Clover, ground 
+- 20 g PTFE, optional 
+- 0.4 g Uranium 
```

Comment:
I couldn't buy this anywhere. Please provide link where I can buy this item for my booms... ...I mean... ...sauce